### PR TITLE
[Snyk] Fix for 2 vulnerabilities

### PR DESCRIPTION
**Pull request by israel-hdez**
<h3>Snyk has created this PR to fix one or more vulnerable packages in the `yarn` dependencies of this project.</h3>

#### Changes included in this PR

- Changes to the following files to upgrade the vulnerable dependencies to a fixed version:
    - package.json
    - yarn.lock

##### With an upgrade:
Severity                   | Priority Score (*)                   | Issue                   | Breaking Change                   | Exploit Maturity
:-------------------------:|-------------------------|:-------------------------|:-------------------------|:-------------------------
medium severity  |  **658/1000**  <br/> **Why?** Proof of Concept exploit, Recently disclosed, Has a fix available, CVSS 5.3  | Regular Expression Denial of Service (ReDoS) <br/>[SNYK-JS-LODASH-1018905](https://snyk.io/vuln/SNYK-JS-LODASH-1018905) |  Yes  | Proof of Concept 
high severity  |  **753/1000**  <br/> **Why?** Proof of Concept exploit, Recently disclosed, Has a fix available, CVSS 7.2  | Command Injection <br/>[SNYK-JS-LODASH-1040724](https://snyk.io/vuln/SNYK-JS-LODASH-1040724) |  Yes  | Proof of Concept 

(*) Note that the real score may have changed since the PR was raised.

Check the changes in this PR to ensure they won't cause issues with your project.

------------

**Note:** *You are seeing this because you or someone else with access to this repository has authorized Snyk to open fix PRs.*

For more information:  
🧐 [View latest project report](https://app.snyk.io/org/kiali/project/1ea38b53-35b0-4db6-b3f9-8d9b308666cf)

🛠 [Adjust project settings](https://app.snyk.io/org/kiali/project/1ea38b53-35b0-4db6-b3f9-8d9b308666cf/settings)

📚 [Read more about Snyk's upgrade and patch logic](https://support.snyk.io/hc/en-us/articles/360003891078-Snyk-patches-to-fix-vulnerabilities)

[//]: # (snyk:metadata:{"prId":"6ad10166-900d-4c20-8589-87b2312ceff2","dependencies":[{"name":"@patternfly/react-charts","from":"5.3.22","to":"6.1.0"},{"name":"lodash","from":"4.17.20","to":"4.17.21"}],"packageManager":"yarn","projectPublicId":"1ea38b53-35b0-4db6-b3f9-8d9b308666cf","projectUrl":"https://app.snyk.io/org/kiali/project/1ea38b53-35b0-4db6-b3f9-8d9b308666cf?utm_source=github&utm_medium=fix-pr","type":"auto","patch":[],"vulns":["SNYK-JS-LODASH-1018905","SNYK-JS-LODASH-1040724"],"upgrade":["SNYK-JS-LODASH-1018905","SNYK-JS-LODASH-1040724"],"isBreakingChange":true,"env":"prod","prType":"fix","templateVariants":["priorityScore"],"priorityScoreList":[658,753]})

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -74,7 +74,7 @@
     "eventemitter3": "4.0.0",
     "js-yaml": "3.13.1",
     "json-beautify": "1.0.1",
-    "lodash": "4.17.20",
+    "lodash": "4.17.21",
```
Comment (lucasponce) [inline package.json:77]: Bump lodash is usually ok.

File: package.json [hunk 2]
```diff
@@ -53,7 +53,7 @@
   },
   "dependencies": {
     "@patternfly/patternfly": "2.71.6",
-    "@patternfly/react-charts": "5.3.22",
+    "@patternfly/react-charts": "6.1.0",
```
Comment (lucasponce) [inline package.json:56]: Bump a major version of react-charts may have more implications.
Have you tested in deep to see if there is no regression in the components that use this library ?

Comment (aljesusg) [reply package.json:56]: About this...I prefer to go a patternfly general update...sometimes react-charts use another lib of patternfly like dependency, we should upgrade them to the "next core release" instead only one I think

Comment (lucasponce) [reply package.json:56]: Yes.
A major upgrade on patternfly can't be changed in a quick PR.
It requires a major test by all sides Dev + QE to be sure that nothing (features and style) are broken.

So, I don't want to open that task yet.

**Review by lucasponce — CHANGES_REQUESTED** [review]
We need to discuss the bump of the react-charts.
I don't think we can bump this as it is.

**Comment by jshaughn** [comment]
> I don't think we can bump this as it is.

@lucasponce +1, agree with @aljesusg and you, that needs to be done as part of a full patternfly update.  I think we probably should schedule that sort of update to be done in a release prior to the one used in SM 2.1.

**Comment by israel-hdez** [comment]
FWIW, I didn't open this PR :-)
Snyk did it. Notice the very first sentence in the main comment: `Snyk has created this PR [...]`

And I agree that we shouldn't bump `react-charts`. Probably, what is important to know from this automatic PR is that there are important library updates.

**Comment by lucasponce** [comment]
Closing this one to avoid noise, we'll discuss potential updates/upgrades in a separated discussion.

**Pull request closed without merging**